Specs/Scene/ImageryLayerSpec.js

```diff
@@ -367,6 +380,41 @@ defineSuite([
         expect(layer.isDestroyed()).toEqual(true);
     });
 
+    it('allows setting texture filter properties', function() {
```

Comment:
@lilleyse are these tests sufficient?  How is coverage?  Should these also do a smokescreen render?